tests/test_inner.py

```diff
@@ -218,18 +232,19 @@ class ExitCompat(WatchedMixIn, ExitPlugin):
 
 
 class Watcher(object):
-    def __init__(self):
+    def __init__(self, rais=False):
```

Comment:
well is not typo, but I can't use keyword, I've changed the name

well RaiseMixIn will raise on run, and won't call watcher, I need to call watcher to "count" and after that raise, so raising in the watcher is actually nice